packages/alfa-css/src/syntax/function.ts

```diff
@@ -144,7 +144,8 @@ export namespace Function {
           return result;
         }
 
-        const [remainder, value] = result.get();
+        // the previous check ensures the result is Ok
```

Comment:
**Thought:** Comparing with what was done for `Option`, it might be that changing the test to `!result.isOk()` would do the correct narrowing.